js/screen/game.js

```diff
@@ -74,6 +77,28 @@ function create () {
     checkTwoPlayers();
   };
 
+  /**
+   * Checks if two players are connected!
+   */
+  function checkTwoPlayers() {
+    const active_players = airconsole.getActivePlayerDeviceIds();
+
+    // Only update if the game didn't have active players.
+    if (active_players.length == 0) {
+      if (connected_controllers.length >= 2) {
+        // Enough controller devices connected to start the game.
+        // Setting the first 2 controllers to active players.
+        airconsole.setActivePlayers(2);
+        document.getElementById("wait").innerHTML = "";
```

Comment:
We can actually remove this and change it so that the render function just displays a message depending on the state.